lib/dotcom/trip_plan/itinerary_groups.ex

```diff
@@ -0,0 +1,145 @@
+defmodule Dotcom.TripPlan.ItineraryGroups do
```

Comment:
Ooooh thanks for the file-rename! It's been bugging me that the file name and the module name were mis-matched!